imperative/python/megengine/functional/tensor.py

```diff
@@ -104,38 +104,35 @@ def full(
     dtype=None,
     device=None,
 ) -> Tensor:
-    r"""Creates a tensor of shape ``shape`` filled with ``value``.
+    r"""Return a tensor of shape ``shape`` filled with ``value``.
 
     Args:
         shape: output tensor shape.
         value: fill value.
-        dtype: output tensor data type. If ``dtype`` is ``None``, the output tensor
+        
+    Keyword args:
+        dtype(:attr:`.Tensor.dtype`): output tensor data type. If ``dtype`` is ``None``, the output tensor
```

Comment:
```suggestion
        dtype (:attr:`.Tensor.dtype`): output tensor data type. If ``dtype`` is ``None``, the output tensor
```

出于排版阅读性要求，这里统一加一个空格。